vignettes/tips_and_tricks.qmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "Tips & Tricks "
+format:
+  html:
+    css: custom.css
+tbl-colwidths: [20,20,50,10]
+---
+
+This page collects miscellaneous tips and tricks for **tinyplot**.
+
+# Legend
+
+## Opacity (`alpha`)
+
+By default, the legend inherits the opacity of the plotted elements. In some cases, this may be undesirable. For example, in the following plot, the legend is too light:
+
+```{r}
+library(tinyplot)
+x = rnorm(1e6)
+y = x + rnorm(1e6)
+z = sample(c("a", "b"), 1e6, replace = TRUE)
+dat = data.frame(x, y, z)
+tinyplot(y ~ x | z, data = dat, alpha = .1, pch = 19)
+```
+
+One solution is to draw our plot in two steps. First, we draw an empty plot with the desired (zero transparency) legend. Second, we add the transparent points on top of the existing canvas.
+
+```{r}
+tinyplot(y ~ x | z, data = dat, pch = 19, empty = TRUE)
+tinyplot_add(empty = FALSE, alpha = .1)
+```
+
+# Labels
+
+## Axis label rotation
+
+
+When category labels are long or overlapping, users may want to rotate them for readability. Unfortunately, `tinyplot` does not currently support rotated axis labels out of the box. 
```

Comment:
Nit: We do support vertical labels (with appropriate spacing) through themes. E.g.

```r
tinytheme("clean", las = 2)
plt(Sepal.Length ~ Species, iris)
```

Maybe we could rephrase slightly:

"`tinyplot` supports fully perpendicular (90°) axis labels via themes, e.g. `tinytheme("clean", las = 2)`. However,  we do not currently provide support for other degrees of label rotation out of the box."